server/api/routers/users.js

```diff
@@ -51,7 +48,7 @@ userRouter.post('/login', async (req, res) => {
         }
         await res.status(200).send(user);
       } else {
-        res.sendStatus(400);
+        res.sendStatus(204);
```

Comment:
200 statuses are successes 
400 statuses are client errors - I think we would want a 404 not found?